.vscode/settings.json

```diff
@@ -18,6 +18,9 @@
     "RUST_BACKTRACE": "1"
   },
   "go.useLanguageServer": true,
+  "go.lintTool": "golangci-lint",
+  "go.lintOnSave": "workspace",
+  "go.buildTags": "secretcli",
```

Comment:
So this actually messes up debug mode, since it tells it to not connect to the rust code. Can you remove this line?